statement.go

```diff
@@ -1004,6 +1008,7 @@ type BeginRoStatement struct {
 	Staleness          time.Duration
 	Timestamp          time.Time
 	Priority           pb.RequestOptions_Priority
+	RequestTag         string
```

Comment:
Similarly this can be simply `Tag string`.

Reply:
fixed.